config/config.php

```diff
@@ -80,5 +80,5 @@
     | to after authorization.
     |
     */
-    'redirect_uri' => env('INFUSIONSOFT_REDIRECT_URI', 'infusionsoft/auth/callback')
```

Comment:
@adriallongarriu this should be left the same

Reply:
It appears that the change was outdated, but the code I commented on is correct.